src/components/CreateVideo.js

```diff
@@ -119,23 +193,269 @@ const CreateVideo = props => {
+                  />
+                </label>
+              </div>
+              <div className="truncation">
+                <label>
+                  Truncation
+                  <div
+                    onMouseEnter={handleTruncateHover}
+                    onMouseLeave={handleTruncateHover}
+                  >
+                    ?
+                  </div>
+                  {truncateHover && (
+                    <div>
+                      Controls the variability of images generated. Max value
+                      yield more variable images, while lower values yield
+                      simpler images with more recognizable, normal-looking
+                      objects.
+                    </div>
+                  )}
+                  <input
+                    type="range"
+                    name="truncation"
+                    min={0.1}
+                    max={1}
+                    step="0.05"
+                    value={videoParams.truncation}
+                    onChange={handleSliderChange}
+                  />
+                </label>
+              </div>
+              <div className="pitch">
+                <label>
+                  Pitch Sensitivity
+                  <div
+                    onMouseEnter={handlePitchHover}
+                    onMouseLeave={handlePitchHover}
+                  >
+                    ?
+                  </div>
+                  {pitchHover && (
+                    <div>
+                      Controls how rapidly the thematic content of the video
+                      will react to changes in pitch.
+                    </div>
+                  )}
+                  <input
+                    type="range"
+                    name="pitch_sensitivity"
+                    min={200}
+                    max={295}
+                    step="5"
+                    value={videoParams.pitch_sensitivity}
+                    onChange={handleSliderChange}
+                  />
+                </label>
+              </div>
+              <div className="tempo">
+                <label>
+                  Tempo Sensitivity
+                  <div
+                    onMouseEnter={handleTempoHover}
+                    onMouseLeave={handleTempoHover}
+                  >
+                    ?
+                  </div>
+                  {tempoHover && (
+                    <div>
+                      Controls how rapidly the overall size, position, and
+                      orientation of objects in the images will react to changes
+                      in volume and tempo.{" "}
+                    </div>
+                  )}
+                  <input
+                    type="range"
+                    name="tempo_sensitivity"
+                    min={0.05}
+                    max={0.8}
+                    step="0.05"
+                    value={videoParams.tempo_sensitivity}
+                    onChange={handleSliderChange}
+                  />
```

Comment:
A next step for these is probably making them their own components and passing down things as props.

Reply:
I like this a lot. I may girl the repo and try to get this worked out over the weekend.